src/isc-dhcp/patch/0007-Uncomment-USE_SOCKETS-to-force-dhcrelay-to-create-an.patch

```diff
@@ -0,0 +1,26 @@
+From f8fc69c18abee41bdea38d0fff4d9c6f65f46884 Mon Sep 17 00:00:00 2001
+From: Joe LeVeque <jolevequ@microsoft.com>
+Date: Sat, 4 May 2019 04:30:42 +0000
+Subject: [PATCH 1/3] Uncomment 'USE_SOCKETS' to force dhcrelay to create and
+ bind separate sockets to each interface
+
+---
+ includes/site.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/includes/site.h b/includes/site.h
+index f11fefb..fd129a1 100644
+--- a/includes/site.h
++++ b/includes/site.h
+@@ -162,7 +162,7 @@
+    the aforementioned problems do not matter to you, or if no other
+    API is supported for your system, you may want to go with it. */
+ 
+-/* #define USE_SOCKETS */
++#define USE_SOCKETS
```

Comment:
>USE_SOCKETS [](start = 9, length = 11)

Use config option instead?
[https://salsa.debian.org/dhcp-team/isc-dhcp/blob/master/configure.ac#L222](https://salsa.debian.org/dhcp-team/isc-dhcp/blob/master/configure.ac#L222)

Reply:
I considered applying this as an option to `configure`, however I felt it was more visible to have all relevant changes in the source code itself. It's easier to understand what's going on.

Reply:
Discussed with Guohan. Agreed to patch debian/rules to add flag. Done in commit https://github.com/Azure/sonic-buildimage/pull/2857/commits/c5be8196a4bce3317538d98e834f5d0dc7bc5082